docs/source/exposing.md

```diff
@@ -0,0 +1,266 @@
+# Exposing ScyllaCluster
+
+This document explains how ScyllaDB Operator exposes ScyllaClusters in different network setups.
+A ScyllaCluster can be exposed in various network configurations, independently to clients and nodes.
+
+``` note::
+ScyllaClusters can be only exposed when the ScyllaDB version used version is `>=2023.1` ScyllaDB Enterprise or `>=5.2` ScyllaDB Open Source.
+```
+
+## Expose Options
+
+`exposeOptions` as name suggests consists of configuration options for exposing ScyllaCluster's.
```

Comment:
```suggestion
`exposeOptions` specifies configuration options for exposing ScyllaClusters.
```
I'd drop the "as name suggests", comes of as a bit aggressive :stuck_out_tongue: